spec/checks/file_content_check_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../spec_helper'
+
+describe RepoAudit::Checks::FileContentCheck do
+  let(:metadata)  { {description: 'check description', style_guide_url: 'www.example.com'} }
+  let(:arguments) { {filename: 'document.txt', content_matchers: content_matchers} }
+  let(:content_matchers) { ['MIT License', 'Copyright \(c\) [\d-]+ Crown copyright \(Ministry of Justice\)'] }
+
+  subject { described_class.new(metadata, arguments) }
+
+  it_behaves_like 'a Check', name: :file_content_check
+
+  context '#run' do
+    let(:repository) { double('Repository', full_name: 'org/repo-name') }
+
+    before do
+      allow(RepoAudit::FileRequestHelper).to receive(:fetch).with(
+        'https://raw.githubusercontent.com/org/repo-name/master/document.txt'
+      ).and_return(file_content)
+    end
+
+    context 'for a success result' do
+      let(:file_content) { File.read('spec/fixtures/example-license.txt') }
+
+      it 'returns the expected result' do
+        expect(subject).to receive(:success)
+        subject.run(repository)
+      end
+    end
+
+    context 'for a failure result' do
```

Comment:
Yep, I will include a test for that scenario. It should fail.

Right now it is all or nothing, as in all the matchers need to be successful for the whole `check` to be successful (i.e. all the strings must be in the content of the file being checked). But if we were to anticipate the necessity of a check that is successful `as long as any of the given matchers are found` then we can extend the check to cope with it or create a new check for that.